cloud/linode/fake_linode_test.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/linode/linodego"
 )
 
+const apiVersion = "v4"
```

Comment:
Looks like you're doing two PRs in one - updating linodego and this firewall attachment stuff - can you then clean up the PRs, please? I was under the impression we'd do these two PRs in succession, but since you're doing both, it'd be good for this to be cleaned up.